checks/libs/win/winpdh.py

```diff
@@ -0,0 +1,49 @@
+import time
+import sys
+import win32pdh
+import psutil
+
+class WinPDHCounter(object):
+    
+    def __init__(self, class_name, instance_name):
+        self._class_name = class_name
+        self._instance_name = instance_name
+
+    def get_single_value(self):
+        hq = win32pdh.OpenQuery()
+        path = win32pdh.MakeCounterPath((None, self._class_name, None, None, 0, self._instance_name))
+        counter_handle = win32pdh.AddCounter(hq, path)
+        win32pdh.CollectQueryData(hq)
+        t, val = win32pdh.GetFormattedCounterValue(counter_handle, win32pdh.PDH_FMT_LARGE)
```

Comment:
`PDH_FMT_LARGE` vs `PDH_FMT_LONG ` (in `get_all_values`). Also https://www.cac.cornell.edu/wiki/index.php?title=Performance_Data_Helper_in_Python_with_win32pdh suggests to use `PDH_FMT_FLOAT`